.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "asciidoc.preview.attributes": {
```

Comment:
Did you mean to commit the vscode settings? I think we shouldn't add them.

Reply:
I did. They are very useful, and they belong to the project. An alternative would be to add `.vscode` to `.gitignore`, but I think it's better this way.

Reply:
That depends how many folks are using VSCode for editing this guide. If it is only you, I think we shouldn't add the settings.

Reply:
I think everybody should use VSCode :)

Reply:
Also note that the settings are for all guides.

Reply:
I disagree that we should add the `.vscode` file, though I won't die on that hill.